cli/cmd/sudo/project/search.go

```diff
@@ -50,6 +57,89 @@ func SearchCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			if status {
+				var table []*projectStatusTableRow
+				ch.Printer.Println()
+				for _, name := range res.Names {
+					org := strings.Split(name, "/")[0]
+					project := strings.Split(name, "/")[1]
+
+					proj, err := client.GetProject(ctx, &adminv1.GetProjectRequest{
```

Comment:
Extract lines 67 to 123 to a util function, something like `func newProjectStatusTableRow(ctx, client, org, project) (*projectStatusTableRow, error)`